ZENKO-3942 - Add test on redis alerts

lavigne958 opened the pull request:
add tests on redis alerts

Commit:
ZENKO-3942 - Add test on redis alerts

bert-e commented:
# Hello lavigne958,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/zenko/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3942 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.3.0`


Please check the `Fix Version/s` of ZENKO-3942, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

jbertran commented on monitoring/redis/alerts.test.yaml:
```diff
@@ -0,0 +1,75 @@
+  - interval: 30s
+    input_series:
+      - series: up{namespace="zenko", service="artesca-data-base-cache-metrics"}
+        values: 1 1 0+0x2
```
You may as well be explicit here :grin:

lavigne958 replied:
done

Commit:
ZENKO-3942 - check alert does not trigger too early

Add extra tests on the alerts to make sure it does not trigger
earlier than expected.

Commit:
ZENKO-3942 - use explicit values for short series

lavigne958 commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/ba595f97502a967d04f97d21dd8549e2453c681a) did not succeed in branch feature/ZENKO-3942_redis_alert_tests.




*The following options are set:* **approve**

francoisferrand commented:
I think this should be targeted at 2.2 branch, right?

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3942 contains:


* `2.2.22`

* `2.3.0`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.3.0`


Please check the `Fix Version/s` of ZENKO-3942, or the target
branch of this pull request.




*The following options are set:* **approve**

bert-e commented:
# Conflict




There is a conflict between your branch `feature/ZENKO-3942_redis_alert_tests` and the
destination branch `development/2.2`.

Please resolve the conflict on **the feature branch** (`feature/ZENKO-3942_redis_alert_tests`).

```sh
git fetch && \
git checkout origin/feature/ZENKO-3942_redis_alert_tests && \
git merge origin/development/2.2
```

Resolve merge conflicts and commit

```sh
git push origin HEAD:feature/ZENKO-3942_redis_alert_tests
```







*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `feature/ZENKO-3942_redis_alert_tests` into
`development/2.2`
* [w/2.3/feature/ZENKO-3942_redis_alert_tests](https://github.com/scality/zenko/compare/development/2.3...w/2.3/feature/ZENKO-3942_redis_alert_tests) will be merged into `development/2.3` (pull request #1509)



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.







*The following options are set:* **approve**

lavigne958 commented:
rebased my branch to it can target dev/2.2

lavigne958 commented:
ping bert-e

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/34afac6f3407c0da215e977277ed7d3fa34f2cd7) did not succeed in branch w/2.3/feature/ZENKO-3942_redis_alert_tests.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`



The following branches have **NOT** changed:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`



Please check the status of the associated issue ZENKO-3942.

Goodbye lavigne958.
